clojure-mode.el

```diff
@@ -1009,7 +1009,7 @@ nil."
                     (one-or-more (any whitespace "\n")))
       ;; why is this here? oh (in-ns 'foo) or (ns+ :user)
       (zero-or-one (any ":'"))
-      (group (one-or-more (not (any "()\"" whitespace))) word-end)))
+      (group (one-or-more (not (any "()\"" whitespace))) symbol-end)))
```

Comment:
AFAICT, that regexp should be just `(1+ (any symbol word))`. But you're the voice of experience here @bbatsov.